server/src/models/gorilla.rs

```diff
@@ -264,3 +281,258 @@ impl<'a> Bits<'a> {
         value as u32
     }
 }
+
+/// Append one or multiple bits to a `vec<u8>`.
+struct BitVecBuilder {
+    current_byte: u8,
```

Comment:
Maybe add doc comments to the struct fields for consistency.

Reply:
Fixed.